esx_service/cli/vmdkops_admin_test.py

```diff
@@ -677,6 +692,7 @@ def test_tenant_vm(self):
         # create tenant1 without adding any vms and privilege
         error_info, tenant = auth_api._tenant_create(
                                                     name=self.tenant1_name,
+                                                    default_datastore=auth_data_const.VM_DS,
```

Comment:
good question. I think i t should show either VM_DS or VM_DS(read_ds_name) so they know where is the DS :-)